src/components/repository-section-title.component.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+import { StateBadge } from 'components';
+import { colors, fonts } from 'config';
+
+type Props = {
+  text: string,
+  open: number,
+  closed: number,
+};
+
+const styles = StyleSheet.create({
+  title: {
+    flexDirection: 'row',
+  },
+  titleText: {
+    color: colors.black,
+    ...fonts.fontPrimaryBold,
+  },
+  badge: {
+    marginLeft: 10,
+  },
+});
+
+export const RepositorySectionTitle = ({ text, open, closed }: Props) => {
```

Comment:
I like this component 🎉  Only thing I can think of is making the props a little more descriptive (`openCount` and `closedCount` for example)